calibration/CTDMOG/CGINS-CTDMOG-13593__20150617.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13593,CC_ptcb1,0.000225,
 37-13593,CC_ptcb2,0,
 37-13593,CC_ptempa0,144.0794,
-37-13593,CC_ptempa1,-0.06115967,
+37-13593,CC_ptempa1,-6.115968e-02,
 37-13593,CC_ptempa2,-1.326181e-06,
-37-13593,CC_wbotc,6.4874e-07,
+37-13593,CC_wbotc,6.487438e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT